pkg/email/templates/trb_advice_letter_internal_review.gohtml

```diff
@@ -0,0 +1,7 @@
+{{template "easi_header.gohtml"}}
+
+<p>{{.TRBLeadName}} has completed a draft advice letter for {{.TRBRequestName}}. This advice letter is now ready for internal review. Please take a moment to look over the draft and make any suggestions for improvement.</p>
```

Comment:
I just synced up with Zoë - It looks like the TRB Lead being assigned is totally optional throughout the entire TRB Request lifecycle. For this reason, I think we'll modify this template to match this updated [Figma](https://www.figma.com/file/iGatgTnlzTuqW9CQFYHDam/TRB-Intake-e2e?node-id=846%3A66293)